packages/e2e-tests/specs/experiments/navigation-editor.test.js

```diff
@@ -324,7 +324,7 @@ describe( 'Navigation editor', () => {
 
 		// Select a link block with nested links in a submenu.
```

Comment:
Should probably update the comment to "submenu block".

Reply:
Thanks, done 👍